docs/developing.rst

```diff
@@ -34,15 +34,6 @@ Rally uses automatic code formatters. They're enforced by ``make lint`` and you
 
 However, consider using editor integrations to do it automatically: you'll need to configure `black <https://black.readthedocs.io/en/stable/integrations/editors.html>`_ and `isort <https://github.com/PyCQA/isort/wiki/isort-Plugins>`_.
 
-Known Issues
```

Comment:
Is there a specific reason we removed the known issue(s)?

Reply:
I mentioned it in the commit, the issue is fixed in pyenv

Reply:
Do'h 🤦 . Thanks for confirming :-)

Reply:
Reading that again, I see that my comment is terse and sounds like it was obvious. It was not, and GitHub's UI makes it difficult to see the commit descriptions. Sorry about that!

Reply:
> Reading that again, I see that my comment is terse and sounds like it was obvious. It was not, and GitHub's UI makes it difficult to see the commit descriptions. Sorry about that!

Haha, not a problem. [I did not assume any malice :- `01.02, /FORMAT` ;-).](https://www.elastic.co/about/our-source-code)